CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 11.14.0
+ - Added SSL settings for: [#1115](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1115)
+   - `ssl_truststore_type`: The format of the truststore file
```

Comment:
Is this PR in the changelog correct?  Shouldn't it be a link to this PR (#1118)?

Reply:
Yes, you're right, it should be #1118. I'll submit a fix! Thanks for the heads up!

Reply:
Thanks, @edmo.  Remember that the link will be wrong for any release notes that include 11.14.0, and will require a manual fix.   :-)